packages/graphback-core/src/db/buildModelTableMap.ts

```diff
@@ -37,15 +37,15 @@ export function getTableName(model: GraphQLObjectType): string {
 
 /**
  * Gets the datase column name for a GraphQL field.
- * Checks for the `@db.name` annotation for a customised name
- * 
- * @param field 
+ * Checks for the `@db(name)` annotation for a customised name
+ *
+ * @param field
  */
 export function getColumnName(field: GraphQLField<any, any>): string {
   let columnName = field.name;
 
   const dbAnnotations = parseDbAnnotations(field);
-  if (dbAnnotations.name) {
+  if (dbAnnotations && dbAnnotations.name) {
```

Comment:
```suggestion
  if (dbAnnotations?.name) {
```

Reply:
Using oportunity. Do we have the same key for the field name and table name?

Reply:
Yep, `@db(name: 'fieldOrTableName')`.